### PR TITLE
Create button Save and change on header

### PR DESCRIPTION
**Pull request by DSsantosss**
Change on Button save on RegistrationScreen and change on header in button and icon bell

### CODE REVIEW

File: src/pages/home/index.jsx [hunk 1]
```diff
@@ -106,21 +106,24 @@ export function App() {
...
+                placeholder="Pesquisar produtos"
                 onChange={(event) => {
                   setSearch(event.currentTarget.value);
                 }}
               />
             </div>
           </div>
           <div className={style["button-icons"]}>
+
+            <div className={style["div-button"]}>
```
Comment (Yan-pg) [inline src/pages/home/index.jsx:117]: It's not indented here, talk to @isacmoreira10 to give you a guide that will indent the code on its own

File: src/pages/registrationScreen/RegistrationScreen.jsx [hunk 2]
```diff
@@ -2,7 +2,11 @@ import { CaretDown, CaretUp, Image } from "@phosphor-icons/react";
 import styles from "../../pages/registrationScreen/RegistrationScreen.module.css";
 import { Sidebar } from "../../components/sidebar";
 import { Input } from "../../components/Input";
+ button-save-style
```
Comment (Yan-pg) [inline src/pages/registrationScreen/RegistrationScreen.jsx]: remove this code

File: src/pages/registrationScreen/RegistrationScreen.jsx [hunk 3]
```diff
@@ -2,7 +2,11 @@ import { CaretDown, CaretUp, Image } from "@phosphor-icons/react";
...
+import { Image } from "@phosphor-icons/react";
+import { Button } from "../../components/Button/Button";
 import { useState } from "react";
+ main
```
Comment (Yan-pg) [inline src/pages/registrationScreen/RegistrationScreen.jsx]: remove this code

File: src/pages/registrationScreen/RegistrationScreen.module.css [hunk 4]
```diff
@@ -121,6 +120,20 @@
   background: #242424;
   resize: none;
 }
+button-save-style
```
Comment (Yan-pg) [inline src/pages/registrationScreen/RegistrationScreen.module.css]: remove this code

File: src/pages/registrationScreen/RegistrationScreen.module.css [hunk 5]
```diff
@@ -176,3 +189,4 @@ input[type="number"]::-webkit-outer-spin-button {
 .icon-caret-down {
   cursor: pointer;
 }
+ main
```
Comment (Yan-pg) [inline src/pages/registrationScreen/RegistrationScreen.module.css]: remove this code

**Review by yeaah-dev — APPROVED** [review]

**Pull request merged**